react/features/filmstrip/components/native/RaisedHandIndicator.js

```diff
@@ -16,6 +16,11 @@ import styles from './styles';
 
 type Props = AbstractProps & {
 
+    /**
+     * Whether or not connection indicator are visible.
+     */
+    connectionIndicatorVisibility: boolean,
```

Comment:
This feels a bit hackish. The raise hand indicator shouldn't need to know if the other indicator is visible in order to render itself. Maybe we can put them in some container? Perhaps looking how the web does it helps?